### DISCUSSION TITLE
Tab Session Manager not keeping names for Chrome's windows upon saving

### DISCUSSION THREAD

**Discussion by susancolvine** [sienori/Tab-Session-Manager#809] [Q&A] [answered]
Hello! So I've been encountering an issue with Tab Session Manager lately. It seems it doesn't register the names for Chrome's windows upon saving sessions, but shows instead the regular "Window" blank title for each of them. 

Going by Tab Session Manager's main page, this doesn't seem normal, as shown below: 

![image](https://user-images.githubusercontent.com/85621780/121349865-fe11f880-c8f7-11eb-9313-7142bdb6fbef.png)
Here is the extension's page seen in Chrome's store, with the window's names as they should normally appear. 
.
.
.
![image](https://user-images.githubusercontent.com/85621780/121350469-bb9ceb80-c8f8-11eb-8c4e-1d5735a2854d.png)
But here is my session's page, when I save it. As you can see, the window's specific names don't show here, no matter how I try to change the settings. I would like to know if there's any way to fix this, or work around this issue? Having distinct names for each saved window is very important to me for telling tab groups apart. 

Thank you!

**Comment by sienori** [comment, marked as answer]
The window title was not displayed in Chrome.
In version 6.9.0, the title of the active tab is now displayed as the window title.
However, the window naming feature of Chrome is not supported.

**Reply by rudimirz** [reply]
> The window title was not displayed in Chrome. In version 6.9.0, the title of the active tab is now displayed as the window title. However, the window naming feature of Chrome is not supported.

Hi, is there an update after 4 months for window naming? that'd be very useful.

**Comment by E-Capp** [comment]
Saving the window name would be very helpful.

**Comment by michael-sorokin** [comment]
I also need Window name saving

**Comment by adamkerz** [comment]
This is why it's not currently possible: https://issues.chromium.org/issues/40174035

**Comment by bandaid** [comment]
no one needs window name saving, unless your intention is exploitive. tabs are sandboxed, tough crackers!

**Comment by howar31** [comment]
uh, having same issue and Googled here.
If saving name is currently not possible, would it be possible to customize the window name in Manager?

**Reply by adamkerz** [reply]
The problem is that Tab Session Manager cannot tell Chrome to set a name for the Window; there is no API that allows it. It's possible that you could set a name that ONLY shows up in the manager window/saved sessions list.

But that's of no value to me, personally. I have 8 windows, each for different work projects and two personal windows and I need to see the name I chose in the task bar to switch to the right window quickly.
